librz/main/rz-ax.c

```diff
@@ -613,7 +647,12 @@ static int rax(RzNum *num, char *str, int len, int last, ut64 *_flags, int *fm)
 
 		return true;
 	} else if (has_flag(flags, RZ_AX_FLAG_BIN_TO_BIGNUM)) { // -L
-		print_hex_from_base2(str);
+		char *temp = rz_ax_parse_bitstring(str);
+		if (!temp)
+			return false;
+		printf(temp);
+		printf("\n");
```

Comment:
brackets, also do not pass temp to printf directly, use `printf("%s\n", temp);`

Reply:
you also forgot to free the buffer

Reply:
Done.